packages/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.ts

```diff
@@ -179,115 +179,57 @@ export class LitNodeClientNodeJs
   createCapacityDelegationAuthSig = async (
     params: CapacityCreditsReq
   ): Promise<CapacityCreditsRes> => {
-    const {
-      dAppOwnerWallet,
-      capacityTokenId,
-      delegateeAddresses,
-      uses,
-      domain,
-      expiration,
-      statement,
-    } = params;
+    // Useful log for debugging
+    if (!params.delegateeAddresses || params.delegateeAddresses.length === 0) {
+      log(
+        `[createCapacityDelegationAuthSig] No delegatee addresses provided. It means that the capability will not restrict access based on delegatee list, but it may still enforce other restrictions such as usage limits (uses) and specific NFT IDs (nft_id).`
+      );
+    }
 
     // -- This is the owner address who holds the Capacity Credits NFT token and wants to delegate its
     // usage to a list of delegatee addresses
     const dAppOwnerWalletAddress = ethers.utils.getAddress(
-      await dAppOwnerWallet.getAddress()
+      await params.dAppOwnerWallet.getAddress()
     );
 
-    // -- default configuration for siwe message unless there are arguments
-    const _domain = domain ?? 'example.com';
-    const _expiration =
-      expiration ?? new Date(Date.now() + 1000 * 60 * 7).toISOString();
-    const _statement = '' ?? statement;
-
-    // -- default configuration for recap object capability
-    const _uses = uses ?? '1';
-
     // -- if it's not ready yet, then connect
     if (!this.ready) {
       await this.connect();
     }
 
     // -- validate
-    if (!dAppOwnerWallet) {
+    if (!params.dAppOwnerWallet) {
```

Comment:
Don't we already assume that it exists above `params.dAppOwnerWallet.getAddress()`?

Reply:
Yea, this check should be on the top instead. It's fixed in another PR